problems/basic_test.go

```diff
@@ -0,0 +1,109 @@
+package problems
+
+import (
+	"context"
+	"crypto/rand"
+	"net/http"
+	"net/url"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/sanity-io/litter"
+	"go.opencensus.io/trace"
+)
+
+func TestBasicProblem_DecorateWithRequest(t *testing.T) {
+	rand.Reader = strings.NewReader("abcdefghijklmnopqrstuvwxyz")
+
+	type fields struct {
```

Comment:
Isn't this just a BasicProblem?

Reply:
I just went with what Goland created for me. Can change to BasicProblem.